2018/002-update-channels.md

```diff
@@ -0,0 +1,130 @@
+ Title
+
+| Field  | Value      |
+|:-------|:-----------|
+| Status | Draft      |
+| Date   | 2018-07-31 |
+
+## Introduction
+
+### Problem description
+
+Updates can be disruptive for a customer's application stack (ex. Kubernetes
+minor upgrade, default pod security policy introduction).
```

Comment:
I think kubernetes minor upgrades should not be disruptive, but kubernetes major upgrades? So from 1.x.y to 1.x.(y+z) should not be disruptive, but 1.x.y to 1.(x+1).z is most likely.